week2.6.4.remember.py

```diff
@@ -0,0 +1,27 @@
+# region remember - this 2
+from PIL import Image
+
+
+def decode_message(path):
+    # Read the image from file
+    img = Image.open(path)

```

Comment:
Instead of explaining each variable I would use a docstring https://swimm.io/blog/code-comments-vs-documentation/